src/scatter_elements.js

```diff
@@ -21,12 +21,12 @@ export function scatterElements(input, indices, updates, {axis = 0} = {}) {
     // output[indices[i, j, k, ...], j, k, ...] = updates[i, j, k, ...] // if axis == 0
     // output[i, indices[i, j, k, ...], k, ...] = updates[i, j, k, ...] // if axis == 1
     // output[i, j, indices[i, j, k, ...], ...] = updates[i, j, k, ...] // if axis == 2
-    const indicesLoc = indices.locationFromIndex(indicesIndex);
-    let indiceValue = indices.getValueByIndex(indicesIndex);
-    indiceValue = indiceValue < 0 ? indiceValue + input.shape[axis] : indiceValue;
-    const outputLoc = indicesLoc.slice();
-    outputLoc[axis] = indiceValue;
-    output.setValueByLocation(outputLoc, updates.getValueByIndex(indicesIndex));
+    const indicesLocation = indices.locationFromIndex(indicesIndex);
+    let indicesValue = indices.getValueByIndex(indicesIndex);
+    indicesValue = indicesValue < 0 ? indicesValue + input.shape[axis] : indicesValue;
+    const outputLocation =
+        [...indicesLocation.slice(0, axis), indicesValue, ...indicesLocation .slice(axis + 1)];
```

Comment:
```suggestion
        [...indicesLocation.slice(0, axis), indicesValue, ...indicesLocation.slice(axis + 1)];
```
Odd space before ` .slice`.

(personally I found the `outputLocation[axis] = indiceValue` less syntactically noisy and less mental effort to understand and validate, but shrug).

Reply:
Good catch! Thanks @fdwr!
Fixed it by #107.